### PR TITLE
Changed background to black.

### PR DESCRIPTION
**Pull request by AMarstonCode**
I changed background to blank in styles.css.
I removed a unneeded simicolon

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -4,8 +4,8 @@
 }
 body {
     font-family: 'PT Serif', serif;
-    font-family: 'Raleway', sans-serif; ;
```
Comment (THalsell) [inline styles.css:7]: why did you remove this?

Comment (AMarstonCode) [reply styles.css:7]: It isnt needed

**Review by THalsell — APPROVED** [review]
